report_async/models/ir_report.py

```diff
@@ -10,6 +10,18 @@
 class Report(models.Model):
     _inherit = "ir.actions.report"
 
+    async_report = fields.Boolean(default=False)
+    async_no_records = fields.Integer(
+        string="Min of Records",
+        default=100,
+        help="Min no of records to use async report functionality; e.g 100+",
+    )
+    async_mail_recipient = fields.Char(
```

Comment:
Just idea, should this be the m2m contacts like in Send Email wizard?

Reply:
the idea was not to have restrict to system users it can be used with emails where the user is not existing on the system. Also the char field is feed directly on the send_mail values as email_to. This allows comma separated emails, so you can just add a list of emails but separated with comma. e.g

`oca@oca.com, tom@oca.com, jane@@usarmy.mil`